File: eform-client/src/app/plugins/modules/vehicles-pn/models/vehicle-pn.model.ts
export class VehiclePnModel {
  id: number;
  contactNumber: string;
  customerName: string;
  brand: string;
  modelName: string;
  registrationDate: Date;
  vinNumber: string;
  contractStartDate: Date;
  contractEndDate: Date;

  constructor() {
    this.contractStartDate = null;
    this.contractEndDate = null;
    this.registrationDate = null;
  }
}

File: eFormAPI/Plugins/Vehicles.Pn/Vehicles.Pn/Infrastructure/Models/VehiclePnModel.cs
using System;

namespace Vehicles.Pn.Infrastructure.Models
{
    public class VehiclePnModel
    {
        public int Id { get; set; }
        public string WorkflowState { get; set; }
        public int? Version { get; set; }
        public DateTime? CreatedAt { get; set; }
        public DateTime? UpdatedAt { get; set; }
        public int CreatedByUserID { get; set; }
        public int UpdatedByUserID { get; set; }
        public string Brand { get; set; }
        public string ModelName { get; set; }
        public DateTime RegistrationDate { get; set; }
        public string VinNumber { get; set; }
        public string PlateNumber { get; set; }
    }
}
